### PR TITLE
Fix infra fo

### PR DESCRIPTION
**Pull request by erebe**

### CODE REVIEW

File: lib/digitalocean/bootstrap/chart_values/nginx-ingress.j2.yaml [hunk 1]
```diff
@@ -218,22 +218,28 @@ controller:
...
-  replicaCount: {{ nginx_minimum_replicas }}
+  # TODO: It should be replaced by tera context values
+  # for now they are now properly injected
+  # see https://github.com/Qovery/engine/pull/264
+  replicaCount: 2
 
   minAvailable: 1
 
+  # TODO: It should be replaced by tera context values
+  # for now they are now properly injected
```
Comment (benjaminch) [inline lib/digitalocean/bootstrap/chart_values/nginx-ingress.j2.yaml:229]: OPT: remove extra word in comment?

File: lib/digitalocean/bootstrap/chart_values/nginx-ingress.j2.yaml [hunk 2]
```diff
@@ -218,22 +218,28 @@ controller:
   ##
   podAnnotations: {}
 
-  replicaCount: {{ nginx_minimum_replicas }}
+  # TODO: It should be replaced by tera context values
+  # for now they are now properly injected
```
Comment (benjaminch) [inline lib/digitalocean/bootstrap/chart_values/nginx-ingress.j2.yaml:222]: OPT: typo / reword?

File: lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml [hunk 3]
```diff
@@ -214,22 +214,28 @@ controller:
   ##
   podAnnotations: {}
 
-  replicaCount: {{ nginx_minimum_replicas }}
+  # TODO: It should be replaced by tera context values
+  # for now they are now properly injected
```
Comment (benjaminch) [inline lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml:218]: OPT: typo / reword?

File: lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml [hunk 4]
```diff
@@ -214,22 +214,28 @@ controller:
...
+  # see https://github.com/Qovery/engine/pull/264
+  replicaCount: 2
 
   minAvailable: 1
 
+  # TODO: It should be replaced by tera context values
+  # for now they are now properly injected
```
Comment (benjaminch) [inline lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml:225]: OPT: typo / reword?

**Review by benjaminch — APPROVED** [review]

File: lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml (hunk 3 above)
Comment (erebe) [reply lib/digitalocean/chart_values/nginx-ingress/nginx-ingress.j2.yaml:218]: done

**Pull request merged**